session.go

```diff
@@ -235,6 +249,13 @@ func (s *session) queueForSend(msg *Message) error {
 
 	s.toSend = append(s.toSend, msgBytes)
 
+	return nil
+}
+
+// queueBatchForSend will validate, persist, and queue a batch of messages for send.
+func (s *session) queueBatchForSend(msgs []*Message) error {
+	s.queueBatchForSendLocked(msgs)
```

Comment:
Needs error handling here

Reply:
Closing this PR, it didn't actually improve performance in our benchmarks

Reply:
@tzachshabtay Can you try this one too with your benchmarks? https://github.com/quickfixgo/quickfix/pull/615